src/components/PageBlocks/Wrap/Card.vue

```diff
@@ -15,12 +15,22 @@
         <div
           v-if="!headerSet"
         >
-          <h5
+          <div
             v-if="block.title"
-            class="text-truncate mb-0"
+            class="d-flex justify-content-between align-content-center"
```

Comment:
We need a more stable approach for the blocks that redefine the slots in header. Since RecordListBase and RecordOrganizer redefine the header.